packages/website/pages/pricing.js

```diff
@@ -53,18 +53,147 @@ const PricingHeader = () => (
     </section>
     <section id="section_trusted_by_logos" className="sectional">
       <div className="grid-middle">
-        <div className="wrapping-logos">
-          <div className="block image-block">
-            <Img src="/images/logos/nft-storage.svg" alt="NFT Storage" />
-            <div className="image-label"></div>
-          </div>
-          <div className="block image-block">
-            <Img src="/images/logos/protocol-labs.svg" alt="Protocol Labs" />
-            <div className="image-label"></div>
-          </div>
-          <div className="block image-block">
-            <Img src="/images/logos/filecoin.svg" alt="Filecoin" />
-            <div className="image-label"></div>
+        <div className="wrapping-logos carousel">
+          <div className="wrapping-logos-inner">
+            <div className="wrapping-logos-images">
+              <div className="block image-block">
+                <Img src="/images/logos/nft-storage.png" alt="NFT Storage" layout="fill" />
```

Comment:
Might be nice if this was driven from an array of logos, but I think this is fine too.

Reply:
great idea actually! bc the set repeats itself so this would be handy.